tests/visualization_tests/test_pareto_front.py

```diff
@@ -193,12 +199,14 @@ def test_get_pareto_front_info_3d(
     )
 
 
+@pytest.mark.filterwarnings("ignore::FutureWarning")
 def test_get_pareto_front_info_invalid_number_of_target_names() -> None:
```

Comment:
Could you remove this annotation? I think this function does not raise `FutureWarning`.